### DISCUSSION TITLE
Application in any namespace

### DISCUSSION THREAD

**Discussion by dkoci** [argoproj/argo-cd#16599] [Q&A]
Hi.
I am following https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace and try to enable application creation in other namespace than argocd. I am running argocd operator in k8s. There were no `argocd-cmd-params-cm` so I created one. I restarted the `argocd-server` and the `argocd-application-controller` but it is not working. What am I missing? I also can not add `--application-namespaces` to the `argocd-server` and the `argocd-application-controller` because they are managed by the operator.

**Comment by crenshaw-dev** [comment]
> it is not working

Can you clarify exactly what you tried to do and how it failed?

**Reply by dkoci** [reply]
I would like to enable application in any namespace. But I cant create argocd application in different namespace than the on in which argcocd is installed. Can someone explain where I can put application.namespace field, because there are no argocd-cmd-params-cm configmap.
